packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -66,6 +66,10 @@ async function main() {
     const credFile: string | undefined = commander.credFile;
     const enableMetrics: boolean = commander.enableMetrics ?? false;
 
+    // We can have some features which only works in df. So choose profile accordingly.
+    if (endpoint === "odsp-df" && profileArg === "ci") {
```

Comment:
this is super specific, as i call out in the package.json, we should just support config overrides for endpoints like we do from drivers